packages/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -66,15 +67,18 @@
 		if (typeof value === 'number' || typeof value === 'bigint' || typeof value === 'boolean')
 			return String(value);
 		if (value instanceof Date) return `'${value.toISOString()}'::TIMESTAMP_MS`;
-		if (Array.isArray(value)) return `[${value.map((x) => jsToDuckDB(x)).join(',')}]`;
+		if (Array.isArray(value)) return `[${value.map((x) => jsToDuckDB(x)).join(', ')}]`;
 		return JSON.stringify(value);
 	}
 
 	function selectedValuesToInput() {
 		let values;
 		if (multiple) {
 			values = $selectedValues.map((x) => x.value);
-			values.toString = () => jsToDuckDB(values);
+			values.toString = () =>
+				values.length === 0
+					? `(select null where 0)`
```

Comment:
what happens here when there are no selected items in a multiselect?

will it throw in a `where category IN ${inputs.my_input.value}` situation?

Reply:
No, I tested it with a few different queries in the CLI and it seemed like `(select null where 0)` was the only that worked consistently
`('')` fails when you're doing something like `select integer_id from table where integer_id in ('')` (from mismatched types)